pytorch3d/io/obj_io.py

```diff
@@ -218,6 +218,9 @@ def _parse_face(
 ):
     face = line.split(" ")[1:]
```

Comment:
I think replacing this line with `face = line.split()[1:]` will do the thing you want without having to add the extra lines below.

Reply:
You are right :)